src/database/hooks/useLocalDatabaseHook.js

```diff
@@ -1,4 +1,5 @@
 import * as React from 'react';
+import _ from 'lodash';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The lodash library is imported but not used in the rest of the code. If it's not needed, consider removing this import to reduce unnecessary dependencies and improve performance.
```diff
- import _ from 'lodash';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->